### ISSUE TITLE
[RC] Hyper-Regularization: An Adaptive Choice for the Learning Rate in Gradient Descent [S1e_ssC5F7]

### ISSUE THREAD

**Issue by reproducibility-org** [reproducibility-challenge/iclr_2019#104]
| Team Name | Affiliation |
|---|---|
| Sharks | epfl; epfl; epfl |

 - Paper: [Hyper-Regularization: An Adaptive Choice for the Learning Rate in Gradient Descent](https://openreview.net/pdf?id=S1e_ssC5F7)
 - Forum: [Comments](https://openreview.net/forum?id=S1e_ssC5F7)
